less/v2/journal.less

```diff
@@ -310,6 +289,15 @@
     flex-grow: 1;
     margin: 5px;
   }
+
+  .space-around {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    clear: both;
+    gap: 10px;
+  }
```

Comment:
We have `.flex-around` provided already by the system, so we should just adjust the journals to use that instead.

```suggestion
```